process/block/metablock.go

```diff
@@ -612,9 +612,11 @@ func (mp *metaProcessor) indexBlock(
 		HeaderHash:             headerHash,
 		Header:                 metaBlock,
 		Body:                   body,
+		PreviousHeader:         lastMetaBlock,
 		RewardsTxs:             rewardsTxs,
 		NotarizedHeadersHashes: notarizedHeadersHashes,
-		PreviousHeader:         lastMetaBlock,
+		HighestFinalBlockNonce: mp.forkDetector.ProbableHighestNonce(),
```

Comment:
GetHighestFinalBlockNonce instead ProbableHighestNonce

Reply:
Right, very good catch, totally missed it.